models.rb

```diff
@@ -36,7 +36,7 @@ class MenuItem
 	property :description,			Text, required: true
 	
 	has n, :reviews
-	has n, :lunch_menu_items
+	has n, :lunch_menu_items, constraint: :destroy
```

Comment:
Are you sure this is the best way to do this?  What about doing it (some other way)?